code/modules/projectiles/guns/beam_rifle.dm

```diff
@@ -472,7 +472,7 @@
 		if(original && (original.layer>=2.75) || ismob(original))
```

Comment:
hmm weird, I would have thought step_to would be entering turfs and causing bump to happen anyway, perhaps it doesn't trigger the Entered call.

Thanks, I need to review this code in more depth I suppose and understand how it's working, and then I might add a comment, and or make 7.75 a define